xforce_connector.py

```diff
@@ -602,7 +604,7 @@ def file_reputation(self, param, action_id):
         connector = XforceConnector()
         connector.print_progress_message = True
         try:
-            ret_val = connector.handle_action(json.dumps(in_json))
+            ret_val = connector._handle_action(json.dumps(in_json), None)
```

Comment:
`_handle_action` is part of the BaseConnector (see [here](https://cd.splunkdev.com/phantom/phantom/-/blob/next/alchemy/pylib/phantom/base_connector.py#L3072))